modules/openapi-generator/src/main/resources/dart2/api_helper.mustache

```diff
@@ -1,6 +1,7 @@
 part of {{pubName}}.api;
 
 const _delimiters = const {'csv': ',', 'ssv': ' ', 'tsv': '\t', 'pipes': '|'};
+final _dateFormatter = DateFormat('yyyy-MM-dd');
```

Comment:
Am I understanding it correctly that _dateFormatter cannot be changed to other date formats during runtime?

Reply:
My thoughts were to protect it against accidentally change. I don't see any reasons to allow modification as the specification says it hast to be this format https://swagger.io/specification/#dataTypes. But if you wish, I can change it

Reply:
Ideally, all REST API should be using the date format following RFC3339 according to OpenAPI spec but there were use cases before in which date using a different format and that's why we make it configurable, e.g. C# configuration class: https://github.com/OpenAPITools/openapi-generator/blob/master/samples/client/petstore/csharp/OpenAPIClient/src/Org.OpenAPITools/Client/Configuration.cs#L100

Reply:
Okay, I will remove the final